src/compiler/php_plugin.cc

```diff
@@ -40,12 +41,25 @@ class PHPGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       return true;
     }
 
+    std::vector<std::pair<grpc::string, grpc::string> > options;
+    ParseGeneratorParameter(parameter, &options);
+
+    grpc::string class_suffix;
+    for (size_t i = 0; i < options.size(); ++i) {
+      if (options[i].first == "class_suffix") {
```

Comment:
If this is a new feature and we have some flexibility, is there any way we can have some kind of protobuf ```grpc_args``` parameter?  It feels really awkward to jam arguments into the ```grpc_out``` parameter.

Might be out of scope for this PR, but just my 2-cents.